pkg/apis/core/v1beta1/constants/types_constants.go

```diff
@@ -65,6 +65,10 @@ const (
 	// the Gardenlet pod.
 	DeploymentNameGardenlet = "gardenlet"
 
+	// DeploymentNameVPNSeedServer is a constant for the name of a Kubernetes deployment object that contains
+	// the vpn-seed-server pod.
+	DeploymentNameVPNSeedServer = "vpn-seed-server"
```

Comment:
Do we need to explicitly state that it's a seed cluster? Can't we just name it `vpn-server` ?

Reply:
After several internal discussions and some confusion, we intentionally wanted to make the names explicit to prevent further confusion. Therefore, we named the server side `vpn-seed-server`. The client secret now also has the verbose, but clear name `vpn-shoot-client`.
I would prefer to keep the names as is.